### PR TITLE
changing to handle-bars

### PR DESCRIPTION
**Pull request by idantb101**
preparing for MVP

### CODE REVIEW

**Comment by NickP123** [comment]
We made a collective decision yesterday to stop with Vue and go to Handlebars. As such we have moved forward with a few responsive pages to get us to MVP.

Should explain some stuff here for QA's. The request:
- pulls in front-end work done by Amir on Advice/Questions/Story pages
- pulls in home page creation and styling done by me. 
- Handlebar work done by Idan

We are now moving to do further work on Handlebars to get us to a better version of our MVP.

**Review by Amirk390 — APPROVED** [review]

**Review by NickP123 — APPROVED** [review]

File: app/src/controllers/router.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  .get(getAllTags)
+  .post(addTag);
+
+router.route('/diseases').get(getAllDiseases);
+
+router.route('/homePagePosts').get(getHomePagePosts);
+
+router.route('/homepage').get(handlebarHomepage);
+
+// router.route('/singlePost').get()
```
Comment (claireinez) [inline app/src/controllers/router.js]: Get rid of this

File: app/src/controllers/tagsHandler.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        'Origin, X-Requested-With, Content-Type, Accept'
+      );
+    }
+  });
+};
+
+const addTag = (req, res, next) => {
+  addTagQuery(req.body, (err, isSuccessful) => {
+    if (err) {
+      // eslint-disable-next-line no-console
```
Comment (claireinez) [inline app/src/controllers/tagsHandler.js]: Unnecessary comment now

**Review by claireinez — CHANGES_REQUESTED** [review]
V minor changes please

**Comment by claireinez** [comment]
Why did you close this PR?

**Comment by idantb101** [comment]
There is a new updated one.

**Pull request closed without merging**